recipe/meta.yaml

```diff
@@ -16,29 +19,32 @@ source:
     folder: cpython-LICENSE
 
 build:
-  number: 0
+  number: 1
   skip: true  # [not win]
   ignore_run_exports_from:
-    - {{ compiler("m2w64_c") }}
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
 
 requirements:
   build:
-    - {{ compiler("m2w64_c") }}
-    - {{ stdlib("m2w64_c") }}
-    - posix
+    - {{ compiler("c") }}
+    - {{ stdlib("c") }}
+    # For bash and friends
+    - m2-base
+  host:
   run:
     # Any Python works as `%PREFIX%/python.exe` is present
     # cli-64.exe (and friends) will look for it as `../python.exe`, so that's
     # why we package the executables in `%PREFIX%/Scripts`
-    - python
+    - python   # [x86_64]
```

Comment:
We are only selecting this so the solver doesn't error out right now, right? But in the future we should re-enable it?

Reply:
yes to both